action_text-trix/Rakefile

```diff
@@ -1,6 +1,13 @@
 require "bundler/gem_tasks"
 require "rake/clean"
 
+begin
+  APP_RAKEFILE = File.expand_path("test/dummy/Rakefile", __dir__)
+  load "rails/tasks/engine.rake" rescue nil
+rescue LoadError
+  # do nothing
+end
```

Comment:
This `rescue` block is in support of the `yarn build-js` command's dependency on executing `rake sync` from the root directory instead of `action_text-trix/`.

Reply:
This feels a bit weird to me, how about this instead?

```suggestion
if defined?(Rails)
  APP_RAKEFILE = File.expand_path("test/dummy/Rakefile", __dir__)
  load "rails/tasks/engine.rake" rescue nil
end
```